### PR TITLE
fix(get-started/about): tweak sentence about layouts

### LINKED ISSUE #885: Content - Getting started
The definitions are a little unclear. I think it would be good to include some "layout" examples, especially since the word isn't fully descriptive.
Xnip2019-04-26_14-18-57

### PR DESCRIPTION
**Pull request by redallen**
https://patternfly-org-pr-2156-v4.surge.sh/v4/get-started/about#layouts

Closes #885

### CODE REVIEW

File: packages/v4/src/content/get-started/about.md [hunk 1]
```diff
@@ -14,7 +14,7 @@ PatternFly is an open source design system created to enable consistency and usa
 Components, like buttons and alerts, can be assembled together to build applications.
 
 ### Layouts
-Layouts are generic tools that allow you to structure and organize the content on your pages.
+Layouts are containers that allow you position components on pages.
```
Comment (abigaeljamie) [inline packages/v4/src/content/get-started/about.md:17]: This looks like a rewrite of the old copy, which existed before the PF home page copy rewrite. The copy we have now is: 

_A layout provides a fully responsive page structure that keeps your components organized and aligned regardless of screen size._ 

So maybe we can add the bit that you have? That would give us: 

_Layouts are containers that allow you to position components on pages. They provide fully responsive page structures that keep your components organized and aligned regardless of screen size._

**Review by abigaeljamie — CHANGES_REQUESTED** [review]

File: packages/v4/src/content/get-started/about.md (hunk 1 above)
Comment (redallen) [reply packages/v4/src/content/get-started/about.md]: I didn't know about this new copy on the homepage. I've updated this PR to use it!

File: packages/v4/src/content/get-started/about.md [hunk 2]
```diff
@@ -26,7 +26,7 @@ Style guidelines define foundational elements of the design system, like color,
 ### Usage and behavior
 Usage and behavior guidelines communicate standards and best practices for common design patterns like navigation, dashboards, or forms.
 
-### Content
+### UX writing
 Content guidelines provide principles and best practices around writing for user experience along with general voice and style guidance.
 
 [**View design guidelines**](/guidelines/colors) <i className="ws-content-blueArrow fas fa-arrow-right pf-u-mx-sm"></i>
```
Comment (abigaeljamie) [inline packages/v4/src/content/get-started/about.md:32]: This link has been updated to this one: https://www.patternfly.org/v4/ux-writing/about

**Review by abigaeljamie — APPROVED** [review]
I just left a quick comment about replacing the UX writing link with the updated style guide. Other than that, all good. Thanks!

**Pull request merged**